en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -181,17 +181,17 @@ def mlp(data):
                              )
 ```
 
-You may curious about why `split(axis=0)` is column cutting. What we need to explain is in OneFlow  `dense` is column storing. Thus the `flow.distribute.split(axis=0)` in above code is column splitting.
+You may curious about why `split(axis=0)` is column cutting. We need to explain is in OneFlow, `dense` is column storage. Thus the `flow.distribute.split(axis=0)` in above code is column splitting.
 
-In addition, `flow.layers.dense`  use `model_distribute`  to set parallelism method. It use the common  `get_variable` to create `blob` in basic level from inner.  Use `get_variable` to config parallelism method called  `distribute`.
+In addition, `flow.layers.dense`  use `model_distribute`  to set parallelism mode, it use the common `get_variable` to create `blob` in basic level from inner, and internally calls the more general interface `get_variable` to create `blob`, while the `get_variable`  interface sets the parallelism mode with the parameter named `distribute`.
```

Comment:
... create `blob`, while the `get_variable`  interface sets the parallelism mode with the parameter named `distribute`.
->
create `blob`. The `get_variable`  interface has a parameter named `distribute` to set the parallelism mode.